packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/web_camera_control_position.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This setting controls how the API handles camera control button on the map
+/// See https://developers.google.com/maps/documentation/javascript/reference/control#ControlPosition for more details.
```

Comment:
Nit: Blank line above this; per Effective Dart style, there should be a blank line after the first sentence in a doc comment.

Reply:
done.